src/components/Molecules/AddressInfo/AddressInfo.tsx

```diff
@@ -0,0 +1,152 @@
+import { Address } from "@/components/Atoms/Address/Address";
+import { Card, CardDescription, CardTitle } from "@/components/ui/card";
+import { Skeleton } from "@/components/ui/skeleton";
+import { GRK_SIZES } from "@/utils/constants/shared.constants";
+import { timestampParser } from "@/utils/functions";
+import { None, Some, type Option } from "@/utils/option";
+import { useCovalent } from "@/utils/store/Covalent";
+import {
+    type CrossChainTransactionsSummary,
+    type AccountInfoProps,
+} from "@/utils/types/molecules.types";
+import { type Chain } from "@covalenthq/client-sdk";
+import { useCallback, useEffect, useState } from "react";
+
+export const AddressInfo: React.FC<AccountInfoProps> = ({
+    address,
+    chain_names,
+}) => {
+    const { covalentClient, chains } = useCovalent();
+
+    const [maybeResult, setResult] =
+        useState<Option<CrossChainTransactionsSummary[]>>(None);
+
+    useEffect(() => {
+        (async () => {
+            setResult(None);
+            const results: CrossChainTransactionsSummary[] = [];
+            await Promise.all(
+                chain_names.map(async (chain_name) => {
```

Comment:
Same as the other comments. Lets keep it to one chain only - we will build the template on a specific chain rather than multi chain

Reply:
To give you an idea - the XYK components are built to take in 1 dex rather than being multi dexed. Try to do the same for these components for one chain

Reply:
Fixed in: caa6216